pkg/ddevapp/app_compose_template.yaml

```
@@ -71,7 +71,7 @@ services:
       - POSTGRES_PASSWORD=db
       - POSTGRES_USER=db
       - POSTGRES_DB=db
-      - PAGER=less -SFX
+      - PAGER=${DDEV_PAGER-less -SFX}
```

This is intentional syntax; the below syntax will not work for an empty string: `export DDEV_PAGER=`, because it will use the default `less -SFX` instead of an empty string:

```suggestion
      - PAGER=${DDEV_PAGER:-less -SFX}
```

---

That fix is right.